taxonomy-editor-frontend/src/pages/editentry/createURL.jsx

```diff
@@ -4,15 +4,29 @@ export function createURL(id) {
 
     // Finding URL to send requests
     let url = API_URL;
-    let isEntry = false;
 
     // ID's can look like: __header__, __footer__, synomym:0, stopword:0
     // For an entry, id looks like en:yogurts
     if (id.startsWith('__header__')) { url += 'header/' }
     else if (id.startsWith('__footer__')) { url += 'footer/' }
     else if (id.startsWith('synonym')) { url += `synonym/${id}/` }
     else if (id.startsWith('stopword')) { url += `stopword/${id}/` }
-    else { url += `entry/${id}/`; isEntry = true; }
+    else { url += `entry/${id}/` }
 
-    return {url, isEntry}
+    return url
+}
+
+export function getIdType(id) {
+
+    // Stores type of ID
+    // Eg: stopword, synonym, entry, header or footer 

```

Comment:
Nice, I like the function 
nit: please turn this into a doc string for the function

Reply:
now you could use it in the `createURL`